components/scream/src/physics/p3/p3_functions_update_prognostics.cpp

```diff
@@ -0,0 +1,15 @@
+#include "p3_functions_update_prognostics_impl.hpp"
+#include "share/scream_types.hpp"
+
+namespace scream {
+  namespace p3 {
```

Comment:
Nitpick: in most of scream, we do not have namespaces increase the indent level.